README.md

```diff
@@ -174,6 +174,7 @@ follow me for update.
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
 I would like to thank microverse for this __wonderful__ opportunity.
+And also to Gregoire Vella on Behance who Original design inspired this idea.
```

Comment:
- [ Required ] As suggested by the previous reviewer I think it would be nice if you give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella), the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).  An appropriate credit should include links to [Gregoire Vella's](https://www.behance.net/gregoirevella) profile on Behance and also a link to [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Currently, your attribution does not include any link. Please the example below as a suggestion on how to achieve this.

## Example
> I would like to give credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella), the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).